tests/test_dle.py

```diff
@@ -318,6 +318,60 @@ def test_structure_2p5D(self):
             assert iters[0].step == par_tile[1]
             assert iters[1].step == par_tile[0]
 
+    
+    @pytest.mark.parametrize('par_tile,expected', [
+        ((32, 4, 4), ((4, 4, 32), (4, 4, 32))),
+        (((16, 4, 4), (16, 16, 16)), ((4, 4, 16), (16, 16, 16))),
+        (((32, 4, 4), 1), ((4, 4, 32), (4, 4, 32))),
+        (((32, 4, 4), 1, 'tag0'), ((4, 4, 32), (4, 4, 32))),
+        ((((32, 4, 8), 1, 'tag0'), ((32, 8, 4), 2)), ((8, 4, 32), (4, 8, 32))),
+    ])
+    def test_openacc_structure(self, par_tile, expected):
+        grid = Grid(shape=(8, 8, 8))
+
+        u = TimeFunction(name="u", grid=grid, space_order=4)
+        v = TimeFunction(name="v", grid=grid, space_order=4)
+
+        eqns = [Eq(u.forward, u.dx),
+                Eq(v.forward, u.forward.dx)]
+
+        op = Operator(eqns, platform='nvidiaX', language='openacc', opt=('advanced', {'par-tile': par_tile,
+                                              'blocklevels': 1, 'blockinner': True}))
+
+        bns, _ = assert_blocking(op, {'x0_blk0', 'x1_blk0'})
+        assert len(bns) == len(expected)
+        for root, v in zip(bns.values(), expected):
+            iters = FindNodes(Iteration).visit(root)
+            iters = [i for i in iters if i.dim.is_Block and i.dim._depth == 1]
+            assert len(iters) == len(v)
+            assert all(i.step == j for i, j in zip(iters, v))
+
+
+    def test_openacc_multi_tile_structure(self):
```

Comment:
how is this test conceptually different than the one above?

Reply:
Indeed the first one was kind of unnecessary. I've dropped it and kept the second one, which is more relevant